.github/workflows/build.yml

```diff
@@ -192,11 +180,32 @@ jobs:
         Add-Content -Path "$Env:GITHUB_ENV" -Value "WINDOWS_CODESIGN_CERTIFICATE_PASSWORD=$Env:WINDOWS_CODESIGN_CERTIFICATE_PASSWORD"
         Add-Content -Path "$Env:GITHUB_ENV" -Value "CMAKE_ARGS_EXTRA=-DWINDOWS_CODESIGN=ON"
 
-    - name: "Set up build environment"
-      id: buildenv
-      run: ${{ matrix.buildenv_script }} setup
-      env:
-        BUILDENV_BASEPATH: ${{ matrix.buildenv_basepath }}
+    - name: Install Nuget
+      uses: nuget/setup-nuget@v1
+      if: runner.os != 'Linux'
+      with:
+        nuget-version: 5.10.0
```

Comment:
Nuget 5.11.0.10 is already installed on GitHub's Windows Server 2019 and Nuget 5.6.0.6489 on macOS 10.15